src/nuster/store/disk.c

```diff
@@ -255,7 +256,6 @@ nst_disk_init(nst_disk_t *disk, hpx_ist_t root, nst_shmem_t *shmem, int clean_te
 #ifdef USE_THREAD
 void *nst_disk_load_thread(void *data) {
     nst_core_t  *core = (nst_core_t *)data;
-
```

Comment:
A blank line as separator. Since disk.c is not changed, just reverting should be fine.